src/lib/redaction.js

```diff
@@ -0,0 +1,12 @@
+const redactDeath = (death, roles) => {
+  return roles.includes('full-details') ? death :
+    [{
+      date: death[0].date,
+      deceased: {
+        forenames: death[0].deceased.forenames,
+        surname: death[0].deceased.surname
+      }
+    }]
```

Comment:
As discussed in the test file. This function should take and return an object not and array.

To run it on an array we can just map it over the array.